webrtc.html

```diff
@@ -1297,6 +1299,25 @@
               </li>
 
               <li>
+                <p>If <var>candidate</var> is <code>null</code>, then run the
+                following steps:</p>
+                <ol>
+                  <li>
+                    <p>For each media section in the last successfully applied
+                    remote description, perform the processing for an
+                    end-of-candidates indication for said media section as
+                    defined in [[TRICKLE-ICE]], unless an ICE restart occurred
+                    for said media section since setting the last remote
+                    description.</p>
```

Comment:
Is ignoring it the right thing to do?  If the local description for the ICE restart is rolled back, we'd still want the end-of-candidates to have applied. 

You already specified that the end-of-candidates applies to the last applied remote description, so that part seems fine.  But perhaps we should specify that it applies to all successfully applied remote descriptions, not just the last one.  That way, if there are two concurrent remote gathering generations, both will be affected by the end-of-candidates.